### PR TITLE
Making it easier to start contributing

### PR DESCRIPTION
**Pull request by EduardGomezEscandell**
As mentioned [here](https://github.com/EduardGomezEscandell/GoWSL/pull/25#discussion_r1072318666), we should make it easy to get started with the repo.

These are the changes to make contribution easier:
- Created script to prepare repository. It installs WSL and Ubuntu, and copies Ubuntu's tarball to `.\images\jammy.tar.gz` so the demo and tests can use it. This way we avoid issues with rootfs URL stability.
- More readable demo: A few printfs and line breaks go a long way.
- Updated contribution guidelines to indicate how to prepare the repo.
- Removed references to jammy to make the repo more future-proof.

### CODE REVIEW

File: examples/demo.go [hunk 1]
```diff
@@ -10,53 +10,54 @@ import (
...
+	fmt.Println("\nRunning async commands")
+	cmd1 := distro.Command(context.Background(), `sleep 3 && cat goodmorning.txt`)
+	cmd1.Start()
+
+	// Launching and waiting for command 2
+	distro.Command(context.Background(), `echo "Hello, world from WSL!" > "goodmorning.txt"`).Run()
+
+	// Waiting for command 1
+	err = cmd1.Wait()
 	if err != nil && !errors.Is(err, &wsl.ExitError{}) {
```
Comment (didrocks) [inline examples/demo.go:50]: Yeah, I know, this is not part of the diff, but while you are cleaning this up, I would nest this time the errors.is check.

Basically, you want to check "was there an error?", and then depending on the type of error, either panic or print + return.

File: prepare-repository.ps1 [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+    winget install --name 'Windows Subsystem for Linux' --silent ${acceptance}
+    if ( ! $? ) { Exit(1) }
+}
+Write-Output "WSL is installed"
+
+# Gettting Ubuntu
+if ( $(Get-AppPackage | Where-Object Name -like 'CanonicalGroupLimited.Ubuntu').Count -eq 0 ) {
+    if (! $(Test-Winget) ) { Exit(1) }
+    Write-Output "Installing Ubuntu"
+    winget install --Id '9PDXGNCFSCZV' --silent ${acceptance}
```
Comment (didrocks) [inline prepare-repository.ps1:52]: I would like as a const (if possible in powershell?) or at least something defined as a global variable on top (easier to spot if we need to change something)

**Review by didrocks — COMMENTED** [review]
Yeah, this is way much better! Just 2 small comments to get to the last mile, but otherwise, very good :)

File: prepare-repository.ps1 (hunk 2 above)
Comment (EduardGomezEscandell) [reply prepare-repository.ps1]: I put it at the top of the file (as top as it'll let me) so it's easier to see.

**Review by didrocks — APPROVED** [review]
All good (apart the stuck CI…)!

**Comment by EduardGomezEscandell** [comment]
Yeah I had to reinstall WSL on the VM, that seems to have fixed it.

**Pull request merged**